base_custom_attributes/__openerp__.py

```diff
@@ -18,31 +18,28 @@
 #   along with this program.  If not, see <http://www.gnu.org/licenses/>.     #
 #                                                                             #
 ###############################################################################
-
-
-
 {
     'name': 'base_custom_attributes',
     'version': '0.1.1',
     'category': 'Generic Modules/Others',
     'license': 'AGPL-3',
-    'description': """This module adds the possibility to easily create custom attributes in any OpenERP business object. See the product_custom_attributes module for instance.
+    'description': """
```

Comment:
As you created a `README.rst`, description should be removed from `__openerp__.py` this way README.rst will be displayed as module description.

You can move this short description in the README file by the way